src/compas/topology/combinatorics.py

```diff
@@ -6,22 +6,29 @@
 from compas.topology import breadth_first_traverse
 
 
-__all__ = [
-    'vertex_coloring',
-    'connected_components',
-]
-
-
 def vertex_coloring(adjacency):
-    """Color the vertices of a network such that no two colors are adjacent.
+    """Color the vertices of a graph such that no two colors are adjacent.
 
     Parameters
     ----------
-    network : compas.datastructures.Network
-        The network object.
+    adjacency : dict[hashable, dict[hashable, None]] or dict[hashable, sequence[hashable]]
+        An adjacency dictionary representing the connectivity of the graph
+        by mapping nodes identifiers to neighbour identifiers.
+        Examples of valid adjacency dicts are
+
+        * ``{0: [1, 2, 3, 4], 1: [0], 2: [0], 3: [0], 4: [0]}``
+        * ``{0: {1: None, 2: None, 3: None, 4: None}, 1: {0: None}, 2: {0: None}, 3: {0: None}, 4: {0: None}}``
+
+    Returns
+    -------
+    dict[hashable, int]
+        A dict mapping each node of the graph to a color number.
 
     Notes
     -----
+    This algorithms works on any data structure that can be interpreted as a graph:
+    a network, a mesh, a volmesh, ...
```

Comment:
maybe a more definite, less elliptical sentence? ;)
```suggestion
    This algorithms works on any data structure that can be interpreted as a graph, e.g. 
    networks, meshes, volmeshes, etc..
```